connection.js

```diff
@@ -650,532 +651,234 @@ class Connection {
-                    // = 30*20 = 600 = 10 minutes period time for 20 stones
-                    this.TimeNumber = (this.TimeNumber * t.stones_per_period);
-
-                    // because of this conversion, we need to recalculate TimeToString
-                    // specific to canadian (time in human readable)
-                    this.TimeToString = timespanToDisplayString(this.TimeNumber);
-                    // but in conn_log, we display requested time by user for all the stones
-                    // example : user "wanted 5 minutes period time for all the 25 stones"
-                    this.TimeNotificationToString = timespanToDisplayString(t.period_time);
-                    this.EndingSentence = ".";
-
-                    // then we can finally have a modified reject message for canadian period time
-                    // example : connSentence + "period time 5 seconds, it is below minimum period time for all the n stones live in byoyomi : 1 minutes"
-                    conn_log(this.ConnSentence + this.TimecontrolSentence + this.TimeNotificationToString + ", it is" + this.ConnBelowAboveSentence + this.MinimumMaximumSentence + this.TimecontrolSentence + this.SpeedString + " in " + this.TimecontrolString);
-                    // example : "minimum periodtime for all the 25 stones for live ranked games is 5 minutes, please increase periodtime for all the 25 stones."
-                    return { reject : true, msg:  `${this.MinimumMaximumSentence} ${this.TimecontrolSentence} ${this.SpeedSentence} ${this.RankedUnrankedGamesIsSentence} ${this.TimeToString}, ${this.IncreaseDecreaseSentence} ${this.TimecontrolSentence} ${this.EndingSentence}` };
+                if ((timecontrolString === "simple") && t.per_move < timeNumber) {
+                    return UHMAEATForPeriodtimesSimpleRejectProcessing();
                 }
             }
 
             // max
-            if ((config.maxperiodtimecorr || config.maxperiodtimecorrranked || config.maxperiodtimecorrunranked) && (this.SpeedString === "correspondence")) {
-                this.MinimumMaximumSentence = "Maximum ";
-                this.SpeedSentence = "for " + this.SpeedString + " ";
-                this.IncreaseDecreaseSentence = ", please reduce ";
-                this.ConnBelowAboveSentence = " above ";
-                this.ConnSentence = user.username + " wanted" + this.MinimumMaximumSentence + " "; // example : "user wanted minimum "
-                if (config.maxperiodtimecorr && !config.maxperiodtimecorrranked && !config.maxperiodtimecorrunranked) {
-                    this.TimeNumber = config.maxperiodtimecorr;
-                    this.TimeToString = timespanToDisplayString(config.maxperiodtimecorr);
-                    this.RankedUnrankedGamesIsSentence = "games is ";
+            if ((config[maxFamilyArray[0]] || config[maxFamilyArray[1]] || config[maxFamilyArray[2]]) && (speedString === blitzLiveCorrConverted)) {
```

Comment:
`(config[maxFamilyArray[0]] || config[maxFamilyArray[1]] || config[maxFamilyArray[2]])`
can be writen shorter:
`maxFamilyArray.some(e => config[e])`